api/v01/entities/vrp_input.rb

```diff
@@ -166,10 +166,10 @@ module VrpConfiguration
 
   params :vrp_request_resolution do
     optional(:duration, type: Integer, allow_blank: false, desc: 'Maximum duration of resolution')
-    optional(:iterations, type: Integer, allow_blank: false, desc: 'Maximum number of iterations (Jsprit only)')
+    optional(:iterations, type: Integer, allow_blank: false, desc: 'DEPRECATED : Maximum number of iterations')
```

Comment:
```suggestion
    optional(:iterations, type: Integer, allow_blank: false, desc: 'DEPRECATED')
```
We used to replace description by 'DEPRECATED' (and, eventually, what it is replaced by)